src/app/components/sidebar/sidebar.component.scss

```diff
@@ -75,3 +75,11 @@
         }
     }
 }
+
+.holdSpace {
+    width: 239.111px;
```

Comment:
Where did this number come from? It's usually very unlikely that you need decimal points in sizes.

Reply:
I got it from the calculated size of the sidebar on the webpage
![image](https://github.com/canvas-gamification/canvas-gamification-ui/assets/79422004/3fced57f-6874-4f98-886e-0ec1f6b4dd23)